makefile.shade

```diff
@@ -667,6 +667,15 @@ var CI_DARWIN_DROP_PATH = '${Environment.GetEnvironmentVariable("CI_DARWIN_DROP_
     }
     nuget-pack packageVersion='${FULL_VERSION}' outputDir='${BUILD_DIR2}' extra='-NoPackageAnalysis -Properties RuntimeNamePrefix=${RUNTIME_NAME_PREFIX}' nugetPath='.nuget/nuget.exe' each='var nuspecFile in Files.Include(Path.Combine(BUILD_DIR2, "*.nuspec"))'
 
+var CI_DEPENDENCIES_PATHS = '${Environment.GetEnvironmentVariable("CI_DEPENDENCIES_PATHS")}'
```

Comment:
It could. I was just following the pattern from https://github.com/aspnet/dnx/pull/2907#discussion-diff-41206792R642